code/game/machinery/computer/dna_console.dm

```diff
@@ -53,17 +53,16 @@
 	else
 		return ..()
 
-/obj/machinery/computer/scan_consolenew/New()
+/obj/machinery/computer/scan_consolenew/Initialize()
 	..()
-
-	spawn(5)
-		for(dir in list(NORTH,EAST,SOUTH,WEST))
-			connected = locate(/obj/machinery/dna_scannernew, get_step(src, dir))
-			if(!isnull(connected))
-				break
-		injectorready = world.time + INJECTOR_TIMEOUT
-		return
-	return
+	return INITIALIZE_HINT_LATELOAD
```

Comment:
This doesn't need late initialization, it's just looking for the presence of the machine so regular Initialize is fine